conans/test/functional/command/export/export_dirty_test.py

```diff
@@ -75,7 +75,7 @@ def test_export_remove(self):
         self.client.run("export . lasote/stable")
         self.assertIn("Source folder is corrupted, forcing removal", self.client.out)
```

Comment:
Here, if `rmdir` fails to remove the source folder, I would expect an exception being raised from the `rmdir` function before the `clean_dirty` one... Up to here the test is ok, but what is it trying to test in the next two lines?